handbook/engineering/README.md

```diff
@@ -416,6 +416,15 @@ When merging a pull request from a community contributor:
 - Thank and congratulate the contributor.
 - Share the merged PR with the team in the #help-promote channel of Fleet Slack to be publicized on social media. Those who contribute to Fleet and are recognized for their contributions often become great champions for the project.
 
+## Changes to Tables' Schema
```

Comment:
Fix to follow Fleet's sentence casing

~~Changes to Tables' Schema~~
Changes to tables' schema

Reply:
Note: Include that the table yml will override current json so be careful on overriding current table schema for other platforms

Reply:
done